<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1747</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="icon_namewidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>270</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(223, 222, 204)
}

</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>40</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="ordermenu_button">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Order</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="salesmenu_button">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Sales</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reportsmenu_button">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Reports</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="predictionmenu_button">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Prediction</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="productinformationmenu_button">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Product Information</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsmenu_button">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color: white;
	text-align: left;
	height:40px;
	border: none;
	border-radius: 10px
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: rgb(223, 222, 204);
	font-weight: bold;
}</string>
           </property>
           <property name="text">
            <string>Settings</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>200</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sign_out_button">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	color: white;
	height:40px;
	border: none;
}

QPushButton:checked {
	background-color: #F5FAFE;
	color: #1F95EF;
	font-weight: bold;
}</string>
         </property>
         <property name="text">
          <string>Sign Out</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="MenuDisplay" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255)</string>
         </property>
         <property name="currentIndex">
          <number>6</number>
         </property>
         <widget class="QWidget" name="welcomepage">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Welcome</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="product_informationpage">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>PRODUCT INFORMATION</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="order_page">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Order</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sales_page">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Sales</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="reports_page">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Reports</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="prediction_page">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Prediciton</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="settings_page">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>771</width>
               <height>91</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
